README.rst

```diff
@@ -33,44 +46,74 @@ Features
 
 * Combine Microsoft Azure quantum simulators with PennyLane's automatic differentiation and optimization.
 
+.. installation-start-inclusion-marker-do-not-remove
 
 Installation
 ============
 
-PennyLane-qsharp requires both PennyLane and the Microsoft Quantum Development Kit. To install the
-Microsoft QDK and IQ#, `see the intruction details <https://docs.microsoft.com/en-us/quantum/install-guide/index?view=qsharp-preview>`_
-provided by Microsoft.
+This plugin requires Python version 3.6 and above, as well as PennyLane
+and Q#. Installation of this plugin, as well as all dependencies, can be done using ``pip``:
+
+   	$ python -m pip install pennylane-qsharp
+
+
+Make sure you are using the Python 3 version of pip.
+
+Alternatively, you can install PennyLane Q# from the source code by navigating to the top directory and running
+::
+
+	$ python setup.py install
 
-Once the QDK is installed, you can install PennyLane-qsharp via ``pip``:
+Dependencies
+~~~~~~~~~~~~
 
-.. code-block:: bash
+.. highlight:: bash
 
-    $ python -m pip install pennylane-qsharp
+PennyLane Q# requires the following libraries be installed:
 
+* `Python <http://python.org/>`_ >=3.6
+* `IQ# <https://docs.microsoft.com/en-us/quantum/install-guide/index?view=qsharp-preview>`_
 
-Getting started
-===============
+as well as the following Python packages:
 
-Once the PennyLane Q# plugin is installed, the provided QDK devices can be accessed straight away in PennyLane.
+* `PennyLane <http://pennylane.readthedocs.io/>`_
+* `Q# <https://docs.microsoft.com/en-us/quantum/install-guide/python?view=qsharp-preview>`_
```

Comment:
No specific versions required for PL or Q#?

Reply:
To be honest, I just took the existing description, and requirements.txt doesn't specify a version for qsharp. @josh146 , would you know?

Reply:
I'll merge this for now and we can add when necessary!